Master add python meh support

jkonecny12 opened the pull request:
Add https://github.com/rhinstaller/python-meh project support.

jkonecny12 committed:
Improve projects copy code

It was a copy paste hell it has to be abstracted.

jkonecny12 committed:
Add support for python-meh side project

VladimirSlavik commented on anaconda_updates/anaconda_updates/releases/__init__.py:
```diff
@@ -53,6 +53,7 @@ def __init__(self, branch_type,
                  pykickstart_args=[],
                  simpleline_args=[],
                  dasbus_args=[],
+                 meh_args=[],
```
Maybe this could use a refactor too? `proj_args={"meh": "blah"}` etc.

jkonecny12 replied:
Fixed by a complete removal of these. They are not used anyway.

jkonecny12 replied:
Quote from IRC:
```
<sbueno> so maybe saying something like: "Encrypt automatically created mount points by default:"
```

VladimirSlavik reviewed (approved):
Looks good to me. Thank you!

jkonecny12 committed:
Remove side project args variables

These were not used and I don't think there is a reason to have them in
releases. Let's remove that completely to simplify code.

VladimirSlavik reviewed (approved):
Even better! Thanks!
